packages/js/README.md

```diff
@@ -41,6 +41,7 @@ Before you're able to use this Matomo Tracker you need to initialize Matomo with
 const MatomoInstance = new window.MatomoTracker({
   urlBase: 'https://LINK.TO.DOMAIN',
   siteId: 3, // optional, default value: `1`
+  userId: '', // optional, default value: `undefined`
```

Comment:
A bit nitpicky, but could you provide an example of an ID, like `UID76903202` instead of an empty string?